modules/password/password.go

```diff
@@ -7,45 +7,58 @@ package password
 import (
 	"crypto/rand"
 	"math/big"
-	"regexp"
+	"strings"
 	"sync"
 
 	"code.gitea.io/gitea/modules/setting"
 )
 
-var matchComplexities = map[string]regexp.Regexp{}
 var matchComplexityOnce sync.Once
```

Comment:
A nit thing, group all the variables will look better.
```
var (
  matchComplexityOnce sync.Once
  validChars string
)
```

Reply:
@lunny done.

Reply:
It seems not?

Reply:
@lunny Sorry, I had pushed to the backport branch. Now I pushed to this branch.